gems/aws-sdk-core/spec/aws/log/formatter_spec.rb

```diff
@@ -24,6 +24,10 @@ def format(pattern, options = {})
         end
 
         it 'provides a :request_params replacement' do
+          file = Tempfile.open('') do |tempfile|
```

Comment:
I'd actually prefer if we test both paths, rather than just replacing with a tempfile. Better way to ensure we could catch regressions with either path. Though given that it is not critical path in this test, you could choose to change one of the two test files, and not the other and I think that would meet the requirement.